server/src/pki_engine/EmbeddedPKIEngine.js

```diff
@@ -936,14 +937,16 @@ class EmbeddedPKIEngine extends PKIEngine {
    * @returns {Validation} validation
    */
   async verifyIntermediateChain (rootCertificate, intermediateChain, code) {
-    if (!intermediateChain) {
+    let body = { intermediateChain: intermediateChain };
+    let firstCertInChain = await PkiService.retrieveFirstChainIntermediateCertificate(body);
+    if (!firstCertInChain) {
       return new Validation(code, false, ValidationCodes.VALID_STATES.NOT_AVAILABLE,
         `No intermediate chain`);
     }
 
-    let { state, output } = await EmbeddedPKIEngine.validateIntermediateChain(rootCertificate, intermediateChain);
+    let { result, output } = await EmbeddedPKIEngine.verifyCertificateSigning(firstCertInChain, rootCertificate, intermediateChain);
```

Comment:
Is the code review requesting to add functionality to modify the third parameter sent to the verifyCertificateSigning where it would be equal to the original intermediate chain contents minus the first certificate?